clientv3/integration/kv_test.go

```diff
@@ -35,8 +35,8 @@ func TestKVPutError(t *testing.T) {
 	defer testutil.AfterTest(t)
 
 	var (
-		maxReqBytes = 1.5 * 1024 * 1024
-		quota       = int64(maxReqBytes * 1.2)
+		maxReqBytes = 1.5 * 1024 * 1024      // hard coded max in v3_server.go
+		quota       = int64(4 * 1024 * 1024) // the grpc max
```

Comment:
OK, the problem is that `1.2 * maxReqBytes` doesn't give enough slack for 64kb pages so it's rejecting based on exceeded space instead of max request size. It should be something like `quota = maxReqBytes + 8 * pagesize` so there's enough space for the initial pages.